docs/applications/data/example_charts.toml

```diff
@@ -0,0 +1,462 @@
+tick_plot = [
```

Comment:
Is there a way to write the spec as a single string?

Reply:
Sure, then we'll just need to split the string by line. But if we go that "far", wouldn't it be even easier to maintain the example charts if we had a folder inside which each file represents a chart?

Reply:
You already did what I meant. I meant each spec should be one string instead of an array. I'll try to be more precise next time I give feedback.